settings.py

```diff
@@ -337,11 +337,15 @@
 SOCIAL_AUTH_PIPELINE_ASSOCIATE_USER = ('social_core.pipeline.social_auth.associate_user',)
 SOCIAL_AUTH_PIPELINE_LOG = ('seqr.utils.social_auth_pipeline.log_signed_in',)
 
+TERRA_API_CACHE_EXPIRE_SECONDS = os.environ.get('TERRA_API_CACHE_EXPIRE_SECONDS', 60)
```

Comment:
can you update `deploy/kubernetes/shared-settings.yaml` with a reasonable value for this (its probably good if its also 60), and update the env section of `deploy/kubernetes/seqr/seqr.gcloud.yaml` to include this variable:
```
 - name: TERRA_API_CACHE_EXPIRE_SECONDS
    value: "{{ TERRA_API_CACHE_EXPIRE_SECONDS }}"
```

Reply:
this still needs doing

Reply:
Kubernetes files are updated now.